targets/TARGET_ARM_FM/TARGET_FVP_MPS2/us_ticker.c

```diff
@@ -61,12 +87,10 @@ void us_ticker_set_interrupt(timestamp_t timestamp)
         us_ticker_init();
     }
 
-    uint32_t delta = timestamp - us_ticker_read();
-    // enable interrupt
-    US_TICKER_TIMER1->TimerControl = 0x0; // disable timer
-    US_TICKER_TIMER1->TimerControl = 0x62; // enable interrupt and set to 32 bit counter and set to periodic mode
-    US_TICKER_TIMER1->TimerLoad = (delta) * 25; //initialise the timer value
-    US_TICKER_TIMER1->TimerControl |= 0x80; //enable timer
+    uint32_t delta =  timestamp - us_ticker_read();
```

Comment:
`delta` should also handle the overflow case. When current time is before overflow and timestamp value is after overflow (`timestamp` < `current time`).

E.g:
```
    const uint32_t now_ticks = us_ticker_read();
    uint32_t delta_ticks =
            timestamp >= now_ticks ? timestamp - now_ticks : (uint32_t)((uint64_t) timestamp + 0xFFFFFFFF - now_ticks + 1);
```